jenkins/scripts/integration_test.sh

```diff
@@ -201,7 +201,7 @@ echo "Running the tests"
 ssh \
   -o StrictHostKeyChecking=no \
   -o UserKnownHostsFile=/dev/null \
-  -o ServerAliveInterval=15 \
+  -o ServerAliveInterval=900 \
```

Comment:
This will make ssh send heartbeat messages every 15 minutes instead of 15 seconds. Usually you want to send these messages a bit more often for this reason.

I think the problem is having low `ServerAliveInterval` combined with `ServerAliveCountMax=10`, which means it'll disconnect if the server hasn't responded in 2.5 minutes. Which it should, since the connection is open...

I'd like to see something like 60 second interval combined with say 20 max. If the SSH server doesn't send ping back in 20 minutes, it is as good as dead anyways.

Reply:
> I'd like to see something like 60 second interval combined with say 20 max. If the SSH server doesn't send ping back in 20 minutes, it is as good as dead anyways.

+1

Reply:
done